app/web/src/features/profile/edit/styles.ts

```diff
@@ -0,0 +1,81 @@
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  avatar: {
+    width: 120,
+    height: 120,
+  },
+  topFormContainer: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    [theme.breakpoints.up("md")]: {
+      flexDirection: "row",
+      margin: theme.spacing(1, 10),
+    },
+    "& .MuiTextField-root": {
+      width: "100%",
+    },
+  },
+  buttonContainer: {
+    position: "fixed",
+    bottom: 0,
+    left: 0,
+    width: "100%",
+    display: "flex",
+    zIndex: 105,
+    backgroundColor: theme.palette.background.paper,
+    borderTop: `1px solid ${theme.palette.divider}`,
+    justifyContent: "center",
+    paddingBottom: theme.spacing(1),
+    paddingTop: theme.spacing(1),
+  },
+  // Everything under the mapbox
+  bottomFormContainer: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    [theme.breakpoints.up("md")]: {
+      margin: theme.spacing(0, 10),
+    },
+    paddingBottom: theme.spacing(5),
```

Comment:
Sorry I misled you, this needs to be a tiny bit more because it blocks slightly on the home tab.

Reply:
The home tab needed bottom padding at different place. Now it should be okay